package.json

```diff
@@ -11,7 +11,7 @@
     "get:current-dir": "node -e 'process.stdout.write(process.cwd());'",
     "build": "yarn nx run-many --target=build --parallel $(yarn get:cpus-number)",
     "build:affected": "yarn nx affected --target=build --parallel $(yarn get:cpus-number) --output-style stream",
-    "build:tools": "yarn nx run-many --target=build --projects=eslint-plugin,dev-tools --parallel $(yarn get:cpus-number)",
+    "build:tools": "yarn nx run-many --target=build --projects=eslint-plugin,dev-tools,workspace --parallel $(yarn get:cpus-number)",
```

Comment:
How did it impact the postinstall (in term of time) for a build without cache?
If I remember the workspace is quite long to build because of its deps 😭 
If it is bad, we should think about having these scripts in build-helpers 😔 .

Can dev-tools be removed of the build packages?

Reply:
When removing dev-tools, running `build:tools --skip-nx-cache` takes 1m0s on my machine
On release 10-next, running the same command take 44s on my machine 
I think it's acceptable

Reply:
I would say that both or not actually but ok 😁